operator/webhook/introute/sync.py

```diff
@@ -291,15 +291,29 @@ def _create_pod_template(parent, labels, integration_image):
                             "port": management_port,
                             "scheme": scheme
                         },
-                        "initialDelaySeconds": 10,
+                        "failureThreshold": 3,
+                        "periodSeconds": 5,
+                        "timeoutSeconds": 3
                     },
                     "readinessProbe": {
                         "httpGet": {
                             "path": "/actuator/health/readiness",
                             "port": management_port,
                             "scheme": scheme
                         },
-                        "initialDelaySeconds": 10,
+                        "failureThreshold": 1,
```

Comment:
Do we want it to fail 3x for failure threshold? I figure you may want readiness checks to be more aggressive than liveness but I don't know if we want to keep it at just 1.

Reply:
I think it would be better to keep it at 1 to minimize the change a service routes 
traffic to a pod that can't accept it.

Reply:
I could also see having at just 1 lead to some false positives. I'll update it to 2